docs/content/en/tasks/upgrade/_index.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "Upgrade K8ssandra"
+title: "Upgrade to K8ssandra Operator"
```

Comment:
We don't yet actually have this ability -- to "upgrade" -- I'd be inclined to think that we should maybe scrap this entire docs-v2 content related to upgrade and create a new section called "migrate" -- then we could document some information about migrating from an existing K8ssandra v1 cluster to a K8ssandra Operator cluster.

What do you think about that?

Reply:
Yeah, I agree. On this tasks-leftnav branch, two things:

1. I'll remove the Tasks > Upgrade topic and check for links to it.
2. In the existing Tasks > Migrate topic, I'll insert a placeholder section near the top, anticipating a Blog post about migrating from an existing K8ssandra v1 cluster to a K8ssandra Operator cluster.